javascript/javascript2/week3/multiPlayer.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>One Player Quiz</title>
+</head>
+<body>
+
+    <main>
+        <section class="quiz-container">
+            <h2 class="player-title">Player 1</h2>
+            <h3 class="quantity-outof">1 out of 10</h3>
+            <div class="point-number-holder">
+                <p class="question-number">Question</p>
+                <p class="point-holder">Point</p>
+            </div>
+            <h2 class="question-title">question</h2>
+            <div class="buttons-holder">
+                <button class="buttons-player1" value="true">True</button>
+                <button class="buttons-player1" value="false">False</button>
+            </div>
+        </section>
+        <section class="quiz-container">
+            <h2 class="player-title">Player 2</h2>
+            <h3 class="quantity-outof">1 out of 10</h3>
+            <div class="point-number-holder">
+                <p class="question-number">Question</p>
+                <p class="point-holder">Point</p>
+            </div>
+            <h2 class="question-title">question</h2>
+            <div class="buttons-holder">
+                <button class="buttons-player2" value="true">True</button>
+                <button class="buttons-player2" value="false">False</button>
+            </div>
+        </section>
+    </main>
```

Comment:
It is a lot better that now you make sure to add as much HTML as you can, and only change their content using JavaScript! Great job! 🎖️